src/include/mca/__mca_internal/single_thread_matrix_calculation.h

```diff
@@ -312,8 +312,6 @@ void transposeSingleThread(const Matrix<T> &a,
 /* Check whether or not rows of matrix are symmetric
```

Comment:
Update this and make it be similar with the pic below:
![image](https://github.com/Kaiser-Yang/matrix-calculation-accelerator/assets/58209855/8868d797-51a3-450e-8a61-d511f9fd7213)